src/View/Helper/FormHelper.php

```diff
@@ -117,9 +117,9 @@ class FormHelper extends Helper
             // Submit input element.
             'inputSubmit' => '<input type="{{type}}"{{attrs}}/>',
             // Container element used by control().
-            'inputContainer' => '<div class="input {{type}}{{required}}">{{content}}</div>',
+            'inputContainer' => '<div class="input {{type}} {{required}}">{{content}}</div>',
```

Comment:
I dont think you should change the whitespace here, it has other side effects beyond your expected change.

Reply:
Agree, this template should not change.